client/app/helpers/optionset.js

```diff
@@ -51,6 +52,7 @@ const OPTIONSET_LOOKUP = {
   project: {
     dcpPublicstatus: DCPPUBLICSTATUS,
     dcpVisibility: PROJECT_VISIBILITY,
+    status: PROJECT_STATUS,
```

Comment:
This is my own fault, but I realize I'm re-aliasing some properties. Like package.status and package.type. I was trying to simplify, but I should have just stuck with the full property name to be consistent. Just a note to self, I may refactor this later